loopflow-computation/src/main/java/com/farao_community/farao/loopflow_computation/LoopFlowComputation.java

```diff
@@ -148,13 +141,13 @@ private static void buildFlowFromNetwork(Network network, Crac crac, Map<Cnec, D
     }
 
     public Map<Country, Double> getRefNetPositionByCountry(Network network) {
-        if (countryNetPositionComputation == null) {
-            countryNetPositionComputation = new CountryNetPositionComputation(network);
+        if (referenceProgram == null) {
+            referenceProgram = ReferenceProgramBuilder.buildReferenceProgram(network);
         }
-        return countryNetPositionComputation.getNetPositions();
+        return referenceProgram.getAllGlobalNetPositions();
     }
```

Comment:
delete this method : 
* not used
* we have        `  requireNonNull(referenceProgram, "referenceProgram should not be null") `in the constructor

Reply:
deleted